brave/browser-laptop
# macOS: end-user postinstall script to parse promo code from installer .pkg filename

Bash script parses promocode e.g. pem001 from filename e.g. /path/to/Brave-Setup-v0.22-pem001.pkg

Fix #12786 

**Depends on #12946 (use pkg installer instead of dmg for macOS release channel)** ✅ now merged

Submitter Checklist:



### Test plan
- Delete all Brave.app's everywhere (/Applications, browser-laptop/Brave*/, ...)
  - the .pkg installer will see these as existing (and moved installations), and upgrade them instead)
- Build package
- Build installer
- Rename installer with `-pem001` at end (before .pkg)
- Run installer

Observe:
- Brave still runs correctly after install
- `promoCode` file is created inside user-data dir with the contents: `pem001`

Reviewer Checklist:


Tests
